core/silkworm/common/endian.cpp

```diff
@@ -0,0 +1,69 @@
+/*
+   Copyright 2021 The Silkworm Authors
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+*/
+
+#include "endian.hpp"
+
+#include <silkworm/common/util.hpp>
+
+namespace silkworm::endian {
+
+Bytes to_big_compact(const uint64_t value) {
+    uint8_t full_be[sizeof(uint64_t)];
```

Comment:
Can't this be static ?

Reply:
Right, I'll make it thread_local to be on the safe side